MMM-Spotify.js

```diff
@@ -13,9 +13,11 @@ Module.register("MMM-Spotify", {
     iconify: "https://code.iconify.design/1/1.0.6/iconify.min.js",
     //iconify: null,
     //When you use this module with `MMM-CalendarExt` or any other `iconify` used modules together, Set this null.
-
     onStart: null,
-    deviceDisplay: "Listening on"
+    deviceDisplay: "Listening on",
+    miniBarAlbum: true,
+    miniBarScroll: true,
+    miniBarLogo: true,
```

Comment:
I would almost add these within another dict

miniBarConfig: {
   album: true,
   scroll: true,
   logo: true,
}

up to you

Reply:
why not :)

will do it